draft-ietf-tls-exported-authenticator.md

```diff
@@ -169,9 +169,7 @@ Certificate, CertificateVerify, and Finished messages.
 If an authenticator request is present, the extensions used to guide the
 construction of these messages are taken from the authenticator request. If
 there is no authenticator request, the extensions are chosen from the TLS
-handshake. That is, the extensions received in a ClientHello (for servers),
-ServerHello (for clients in TLS 1.2), or EncryptedExtensions (for clients in
-TLS 1.3).
+handshake. That is, the extensions received in a ClientHello (for servers).
```

Comment:
This is wordy, but...

A server can provide an authenticator without a corresponding request, in which case the extensions in the ClientHello are used to determine what extensions are permitted in the Certificate message.

Reply:
That works for me. I rephrased it a bit.